velox/docs/functions/spark/string.rst

```diff
@@ -20,6 +20,24 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
     If ``n < 0``, the result is an empty string.
     If ``n >= 256``, the result is equivalent to chr(``n % 256``).
 
+.. spark:function:: concat_ws(separator, [string]/[array<string>], ...) -> varchar
+
+   Returns the concatenation for ``string`` and all elements in ``array<string>``, separated
+   by ``separator``. ``separator`` can be an empty string. It can take variable number of
+   remaining arguments, where ``string`` & ``array<string>`` can be used in combination. NULL
+   element is skipped in the concatenation. If ``separator`` is NULL, returns NULL, regardless
+   of the following inputs. For non-NULL ``separator``, if no remaining input or all remaining
+   inputs are NULL, returns an empty string. ::
+
+        SELECT concat_ws('~', 'a', 'b', 'c'); -- 'a~b~c'
```

Comment:
How does concat_ws behave when element is empty string or array like `concat_ws('~', 'a', 'b', '')`, or `concat_ws('~', [], ['d'])`?